packages/govern-subgraph/schema.graphql

```diff
@@ -8,46 +8,39 @@ type ERC3000Registry @entity {
 type OptimisticGame @entity {
   id: ID!
   name: String!
+<<<<<<< HEAD
   queue: GovernQueue!
+=======
+>>>>>>> 2da21a6... Subgraph: refactor entities and include review suggestions
   executor: Govern!
 }
 
 type Govern @entity {
   id: ID!
   address: Bytes!
   metadata: Bytes
-  executions: [Execution!]
   games: [OptimisticGame!] @derivedFrom(field: "executor")
+  executions: [Execution!]
   roles: [Role!]
 }
 
 type Execution @entity {
   id: ID!
-  executor: Govern!
   queue: GovernQueue!
-  failureMap: Bytes!
-  actions: [Action!] @derivedFrom(field: "execution")
+  packet: QueuePacket!
   results: [Bytes!]
-}
-
-type Action @entity {
-  id: ID!
-  to: Bytes!
-  value: BigInt!
-  data: Bytes!
-  execution: Execution!
-  queueItem: QueueItem!
+  createdAt: BigInt!
 }
 
 type GovernQueue @entity {
   id: ID!
   address: Bytes!
   config: Config
-  queued: [QueueItem!]
-  challenges: [Challenge!]
-  vetos: [Veto!]
-  executions: [Execution!] @derivedFrom(field: "queue")
   games: [OptimisticGame!] @derivedFrom(field: "queue")
+  scheduledPackets: [Schedule!]
+  challengedPackets: [Challenge!]
+  vetoedPackets: [Veto!]
+  executedPackets: [Execution!] @derivedFrom(field: "queue")
```

Comment:
I find it a bit odd that these are named `somethingPackets`, but don’t point to actual packets (as in `QueuePacket`), but to wrappers instead. Should we just stick to `scheduled`, `challenged`, etc. to better reflect the types?

Reply:
Also proposed this on #82; I think it would make sense to simplify the schema further to be closer to actual types sent by the contract.

Reply:
Make sense, I was trying to reflect that the `packets` are the things being handle. But depending on the type you also have extra information related to that action.